enterprise/backend/test/metabase_enterprise/advanced_permissions/common_test.clj

```diff
@@ -507,7 +507,14 @@
           (with-all-users-data-perms {(mt/id) {:data       {:schemas :block :native :none}
                                                :data-model {:schemas {"PUBLIC" {(mt/id :venues) :all}}}}}
             (mt/user-http-request :rasta :post 200 (format "table/%d/rescan_values" (mt/id :venues)))))
-        (is (= [1 2 3 4] (t2/select-one-fn :values FieldValues, :field_id (mt/id :venues :price))))))
+        (is (= [1 2 3 4] (t2/select-one-fn :values FieldValues, :field_id (mt/id :venues :price)))))
+
+      (testing "An audit log entry is generated when a manually triggered re-scan occurs"
+        (mt/with-model-cleanup [:model/AuditLog :model/Activity]
```

Comment:
In general I think it's OK for tests to write to the audit log without cleanup, right? Lots of other tests will also do things that implicitly trigger audit events to be written and we can't expect every developer to clean up the table.

Reply:
Yeah, It is totally okay for tests to write to the audit log and _not_ clean them up. Maybe most tests will touch the audit log at some point. I don't expect anyone to clean them up in tests.

I see about 29 spots where we have `mt/with-model-cleanup.*:model/AuditLog`. Do you think should we actually just remove them? Reading through, some delete audit logs that would point to nonexistant things. But that isn't actually a problem either, afaik.

Reply:
If so let's get that into a followup since its not an urgent change. I will come back to it once we have auditv2 shipped.